client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -624,7 +787,6 @@ export const useCommandsArrayWpcom = ( {
 		{
 			name: 'addSubscribers',
 			label: __( 'Add subscribers' ),
-			searchLabel: __( 'Import subscribers' ),
```

Comment:
The linked page allows uploading CSV to import subscribers. What do you think about adding keywords with 'import' and 'upload' actions, too?

Reply:
Yep — why removal?